src/app/services/news.service.ts

```diff
@@ -0,0 +1,57 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable, from } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { News } from '../models/News';
+
+export interface NewsService {
```

Comment:
Should we change our approach to naming interfaces? 
Since its not directly clear from the name that 'NewsService' is an interface, maybe we want to go with the C# naming convention by having all interfaces begin with a capital 'I'?

I'll have to change this myself too in my other pull request if we think its useful to do.

Reply:
I'm happy to go either way, but keep in mind that angular style guidelines discourage the use of leading `I`
https://v2.angular.io/docs/ts/latest/guide/style-guide.html#!#naming
however the reason they mention that is because of TypeScript guidelines, which are the guidelines for contributing TO the TypeScript project and not general TypeScript style guides.

 ¯\_(ツ)_/¯

Reply:
ah probably just leave them as is then. And come back to it if it becomes an issue later